src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task06.java

```diff
@@ -39,8 +39,21 @@ public static void main(String[] args) {
      * @return 'Player 1: ${0}. Player 2: ${1}', где ${0} - очки первого игрока, ${1} - очки второго игрока.
      */
     static String getResult(int tanksKilledFirstPlayer, int tanksKilledSecondPlayer) {
+        int totalScorePlayer1 = 0;
+        int totalScorePlayer2 = 0;
+        if (tanksKilledFirstPlayer > 0) {
+            totalScorePlayer1 = tanksKilledFirstPlayer * KILL_POINT;
+        }
+        if (tanksKilledSecondPlayer > 0){
+            totalScorePlayer2 = tanksKilledSecondPlayer * KILL_POINT;
+        }
+        if (tanksKilledFirstPlayer > tanksKilledSecondPlayer) {
+            totalScorePlayer1 = totalScorePlayer1 + BONUS_POINT;
+        } else if (tanksKilledSecondPlayer > tanksKilledFirstPlayer) {
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"